maquetado/scss/login.scss

```diff
@@ -0,0 +1,76 @@
+.login-container {
+  background-color: $wild-sand;
+  display: flex;
+  flex-flow: column wrap;
+  height: 100%;
+  justify-content: center;
+  max-height: 400px;
+  max-width: 300px;
+  width: 100%;
+
+  .login-top {
+    background-color: $cerulean;
+    border: 2px solid $cerulean;
+    height: 100%;
+    max-height: 8px;
+    width: 100%;
+  }
+  
+  .login-bottom {
+    padding: 20px 24px 24px;
+
+    .wolox-logo {
+      height: 70px;
+      max-height: 100%;
+      max-width: 212.42px;
+      width: 100%;
+    }
+
+    .login-form {
+      display: flex;
+      flex-flow: column wrap;
+      flex-grow: 1;
+      width: 100%;
+
+      .login-label {
+        font-size: $font-small;
+        margin-bottom: 8px;
+      } 
+
+      .input-form {
+        background-color: $white;
+        border-radius: 5px;
+        font-size: $font-small;
+        height: 100%;
+        margin-bottom: 16px;
+        max-height: 32px;
+        max-width: 252px;
+        padding: 8px 0;
+        width: 100%;
+      }
+
+      .login-button {
+        background-color: $sulu-green;
+        border-radius: 5px;
+        color: $wild-sand;
+        font-size: $font-small;
+        margin-top: 8px;
+        padding: 8px 0;
+      }
+    }
+
+    .bottom-decorator {
```

Comment:
This border can be add like border-bottom to login-form class

Reply:
You say inside of the form class? There are any criteria for decide if a decorator (or another component that are not a input of that form) must are inside of the form or outside?

Reply:
Done!